source/rfc/metrics.rst

```diff
@@ -0,0 +1,83 @@
+Metrics
+=======
+
+Problem
+-------
+
+Zuul jobs are run as builds on independent, created on-demand, nodes. No information is available about the
+state changes of the nodes throughout the whole buildset run in terms of CPU, memory, storage usage
+or any other useful information related to the buildset itself for that matter. The only results coming
+out of buildsets currently are test reports, run logs and built artifacts.
+
+To have better understanding and control over the CI system and any insight on how changes in code
+impact the building and testing processes, more information about the system the buildsets are being executed on
+needs to be gathered and on the outputs of the builds themselves.
+
+Metrics
+-------
+
+This section contains proposals of metrics to gather during and from buildsets.
+
+#. General
+
+   * run duration
+
+#. Physical resources
```

Comment:
Add examples: peak disk usage, peak memory usage, CPU hours.

Reply:
Gathering peaks is a way to avoid gathering timeseries data, so it can be mentioned as an idea.

Reply:
Added to the list.